diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index 3f070a3..a110c7f 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -106,6 +106,7 @@ exports.networks = [
     { chainId: 4919n, shortName: 'xvm' },
     { chainId: 5000n, shortName: 'mantle' },
     { chainId: 5001n, shortName: 'mantle-testnet' },
+    { chainId: 5003n, shortName: 'mnt-sep' },
     { chainId: 5700n, shortName: 'tsys' },
     { chainId: 6102n, shortName: 'cascadia' },
     { chainId: 7001n, shortName: 'zetachain-athens' },
